articles/install-guide/update.md

```diff
@@ -14,67 +14,102 @@ Learn how to update the Microsoft Quantum Development Kit (QDK) to the latest ve
 
 This article assumes that you already have the QDK installed. If you are installing for the first time, then please refer to the [installation guide](xref:microsoft.quantum.install).
 
+If you saw that a new version of the QDK was released and you would like to get with the times, then you're in the right place.
+The process consists of two parts:
+1. updating your existing Q# files and projects to align your code with any updated syntax
+2. updating the QDK itself for your chosen development environment 
 
-## Updating Q# Projects 
+## 1. Updating Q# Projects 
```

Comment:
```suggestion
## Updating Q# Projects 
```

Reply:
@bromeg this one I made numbered to align with the two "parts" of the update process that are described at the very beginning:
1. Updating Q# projects
2. Updating the QDK

Personally I think it's beneficial to keep them as numbered H2's, but just let me know what you think (if we go no-numbers, I'll remove the number from the "Update QDK" H2 as well)

Reply:
Also deleting the number from the second H2  👍